File: Check_if_the_Sentence_Is_Pangram.go
package main

import "fmt"

func main() {
	sentence := "thequickbrownfoxjumpsoverthelazydog"
	fmt.Println(checkIfPangram(sentence))
}

func checkIfPangram(sentence string) bool {
	mk := make(map[string]bool)
	for _, s := range sentence {
		mk[string(s)] = true
	}
	if len(mk) == 26{
		return true
	}
	return false
}

File: Check_if_the_Sentence_Is_Pangram.rs
use std::collections::HashMap;
fn check_if_pangram(sentence:&str) -> bool{
    let mut hashmp:HashMap<String,bool> = HashMap::new();
    for s in sentence.chars(){
        hashmp.insert(s.to_string(),true);
    }
    if hashmp.len() == 26{
        return true;
    }
    return false;

}

fn main(){
    let sentence = "thequickbrownfoxjumpsoverthelazydog";
    println!("{:?} ",check_if_pangram(&sentence));

}

